tests/store/_unity_catalog/model_registry/test_unity_catalog_rest_store.py

```diff
@@ -356,7 +398,7 @@ def _assert_create_model_version_endpoints_called(
         )
 
 
-def test_create_model_version_aws(store, tmp_path):
+def test_create_model_version_aws(store, local_model_dir):
```

Comment:
To test the happy path (model signature is present), run the existing tests for creating model versions against a valid model version directory with an MLmodel file containing a `signature`